cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -117,21 +119,18 @@ describe(["@tier2"], "Application inventory filter validations", function () {
     it("Name filter validations", function () {
         Application.open();
 
-        // Enter an existing name substring and assert
-        var validSearchInput = applicationsList[0].name.substring(0, 11);
-        applySearchFilter(name, validSearchInput);
+        applySearchFilter(name, applicationsList[0].name);
         cy.wait(2000);
         exists(applicationsList[0].name);
+        notExists(applicationsList[1].name);
 
-        if (applicationsList[1].name.indexOf(validSearchInput) >= 0) {
-            exists(applicationsList[1].name);
-        }
```

Comment:
You shouldn't remove this validation. It checks, that if part of existing name was used as filter string - respective app was found

Reply:
Hi @Neilhamza,I believe the test should validate that both applications are filtered and displayed on the page. While validating just one application might make sense, it's safer to filter more options and test the functionality thoroughly. Therefore, removing the validation doesn't seem reasonable to me too.

Reply:
this test is now irrelevant since  the change of the filter now does not automatically show applications -> there is a need to click on which application i am looking for when filtering
i reverted the code to original (this part of the code will never be true)
should i keep it as i changed? or as it is now (after applying your requested changes)
@kpunwatk @ibragins

Reply:
**even if the statement true the test will fail since i didn't select this app when filtering

Reply:
Look, if we have several matches when you selected checkboxes of several apps - it is absolutely valid scenario as well. This is what I as working on in Issues and Dependencies. For example, you have 5 apps, you selected 2 of them in filter - you need to make sure that both are in the list. Make sense? Please work on implementing this, may be as a separate PR as it is a new TC.

Reply:
sure, ill implement it in a new PR

Reply:
Got it, Even i see we do filter applicationsList[1].name in second part, so not a concern.
PR Looks good to me :+1: